src/sql/tests/parameters.rs

```diff
@@ -17,7 +17,7 @@ use sql::Session;
 #[test]
 fn test_parameter_type_inference() -> Result<(), Box<dyn Error>> {
     let catalog = DummyCatalog;
-    let session = Session::default();
+    let session = Session::new(0);
```

Comment:
In cases like this, I like to introduce a `dummy` method. (And hey, look at that! The catalog is an example of this right here.) Removing the default implementation is exactly right, since there is no longer a useful default for most situations but `Session::new(0)` leaves the reader wondering what the `0` is. Something like `Session::dummy()` solves this nicely; it makes the reader wonder about whether a dummy session is correct or whether it should be a real session, but you don't have to think about the individual parameters.